src/Illuminate/Log/ParsesLogConfiguration.php

```diff
@@ -57,10 +57,6 @@ protected function level(array $config)
      */
     protected function parseChannel(array $config)
     {
-        if (! isset($config['name'])) {
-            return $this->getFallbackChannelName();
-        }
-
-        return $config['name'];
+        return $config['name'] ?? $this->getFallbackChannelName();
```

Comment:
its not a valid if statment

here is the sample one
```
$variableValue = 10;
return ($variableValue > 2 ? true : false); // returns true
```

thats the tests cases are failed